package.json

```diff
@@ -53,6 +54,7 @@
     "yup": "^1.6.1"
   },
   "devDependencies": {
+    "@percy/cli": "^1.31.5",
```

Comment:
damn, lots of packages have "^" in their version, that's quite risky. Something can fail and it would be quite difficult to debug in case some package just upgrades on its own after some deployment. Can I ask you to fixate all of them please (or at least the Percy one)?

Reply:
ok good enough, we'll revisit this later